GLControl/MainForm.cs

```diff
@@ -33,7 +33,8 @@ public partial class MainForm : Form
             4, 3, 0
         };
 
-        // We store the loaded state in a boolean to prevent GL specific instructions to be called by WinForms events.
+        // We store the loaded state in a boolean to prevent GL specific instructions
+        // to be called by Windows Form events before creating the GL context.
```

Comment:
Minor grammar. Should be "We store the loaded state in a boolean to prevent GL-specific instructions being called before the GL context is created."